### PR TITLE
fix upercase in 2009courchesne

### PR DESCRIPTION
**Pull request by nicobou**

### CODE REVIEW

File: metalab-papers.bib [hunk 1]
```diff
@@ -32,7 +32,7 @@ @Misc            {2005husband1
 
 @InBook           {2009courchesne,
   author        = {Luc Courchesne},
-  editor        = {D.A.P./Distributed Art Publishers},
+  editor        = {{D.A.P.}/Distributed Art Publishers},
```
Comment (nicobou) [inline metalab-papers.bib]: This is done auto, no need to do that

File: metalab-papers.bib [hunk 2]
```diff
@@ -32,7 +32,7 @@ @Misc            {2005husband1
 
 @InBook           {2009courchesne,
   author        = {Luc Courchesne},
-  editor        = {D.A.P./Distributed Art Publishers},
```
Comment (nicobou) [inline metalab-papers.bib]: ji fe rutre

File: metalab-papers.bib (hunk 1 above)
Comment (nicobou) [inline metalab-papers.bib]: nop, depends on stuffs

Comment (eldaranne) [inline metalab-papers.bib]: good stuff.

**Pull request merged**